docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: explain_debug
+- Status: in-progress
+- Start Date: 2015-08-10
+- RFC PR: #2047
+- Cockroach Issue:
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
+will cause simple (single table or index) select statements to display the
+underlying key-value pairs that were accessed:
```

Comment:
One more thought: we may have a whole family of debug statements: this one could be `DEBUG KEYS SELECT...`. Another one I anticipate is `DEBUG NODES SELECT...`, which would return information about which nodes were contacted such as leader/follower status (for when we support reads from followers and need to be able to verify that this is happening).